
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Zuolin Cheng's Website</title>
    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
    
    <style>
        body {
            font-size: 16px;
            background-color: black;
            color: white;
        }
        
        .navbar {
            background-color: #222;
            padding: 0px 0px;
        }
        .navbar-collapse {
            display: flex;
            align-items: center;
        }
        .navbar-header {
            flex-grow: 1;
            display: flex;
            align-items: center;
        }
        .navbar-header h1 {
            font-size: 30px;
            color: #aaa;
            margin: 0;
            padding: 18px 0px;
        }
		.navbar-nav>li>a {padding-top: 0px;padding-bottom: 0px;line-height: 60px;font-size: 18px;}
        .navbar-nav-head {
            display: flex;
            margin: 0;
            padding: 0;
        }
        .navbar-nav-head>li {
            list-style: none;
        }
        .navbar-nav-head>li>a {
            padding: 0px;
            padding-left: 12px;
            padding-right: 12px;
            
            font-size: 18px;
            color: #aaa;
            text-decoration: none;
        }		
		
		
		
		a {color: #FFD54F;text-decoration: none;}
		a:visited {color: #B47118;}
		
        .container {
            padding-top: 80px;
            font-size: 18px;
        }
        .container_head {
            display: flex;
            justify-content: space-between;
            align-items: center;
        }
        .intro-content {
            display: flex;
            flex-wrap: nowrap;
            align-items: flex-start;
            color: white;
        }
        .intro-image {
            flex: 0 0 27%;
            padding-right: 25px;
        }
        .intro-image img {
            width: 100%;
            height: auto;
            display: block;
            margin-top: 0;
        }
        .intro-description {
            flex: 0 0 73%;
            margin-top: 0;
            font-size: 18px;
        }
        .intro-description p:first-child {
            margin-top: 0;
        }

        /* Add this for scroll margin */
        #SelectedProjects, #Publications, #Awards{
            scroll-margin-top: 60px; /* Adjust this value to match your navbar height */
        }
        #News, #OngoingProjects{
            scroll-margin-top: 90px; /* Adjust this value to match your navbar height */
        }     
    </style>
</head>



<!-- <body id="home" data-spy="scroll" data-target=".navbar" style="background-color:black;"> -->
<body id="home" data-spy="scroll" data-target=".navbar">


<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">  
	<div class="container" style="padding-top: 0px; padding-bottom: 0px; line-height: 30px; font-size: 18px">    
        <div class="container_head">
            <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
    			<span class="sr-only">Toggle navigation</span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
      		</button>
                <h1>Zuolin Cheng</h1>
            </div>
            	<ul class="nav navbar-nav-head navbar-right">
                	<li><a href="#">Home</a></li>
                	<li><a href="#News">News</a></li> 
                	<li><a href="#OngoingProjects">Ongoing projects</a></li> 
                	<li><a href="#SelectedProjects">Selected projects</a></li>
                	<li><a href="#Publications"">Publications</a></li>
                	<li><a href="#Awards" style = "padding-right: 0px">Awards</a></li>
                	<!-- <li><a href="#Contact">Contact</a></li> -->
                	<!-- <li><a href="https://ccwang92.github.io/#News">News</a></li> -->
                	<!-- <li><a href="http://oliviawang90.github.io/~yj9xs/#Awards">Awards</a></li> -->
                	<!-- <li><a href="https://oliviawang90.github.io/#Projects">Onging Project</a></li> -->
                	<!-- <li><a href="https://oliviawang90.github.io/#Education">Education</a></li> -->
            	</ul>
        </div>
    </div>
</nav>





</br></br></br>



	
	
<!--  <div id="Introduction" class="page-header container">
	<h1>About Me </h1>
</div> -->
<div class="container" style="padding-top: 80px; font-size: 18px">
<!-- <div class="container"; font-size: 18px"> -->
	<div class="intro-container">
	<div class="intro-content">
		<div class="intro-image">
			<img class="img-responsive img-rounded" src="data/imgs/me2.jpg" alt="">  <!-- Image -->
		</div>
		
		<div class="intro-description" align="justify">
		<p style="color:white;">
		Hello! Welcome to my portfolio! I am Zuolin Cheng, a bioinformatician with a Ph.D. from Virginia Tech, USA, and a master's from Peking University, China. 						<!--Before comming to Virginia Tech, I earned my Bachelor's degree from Tsinghua University, Beijing. My undergraduate thesis was about traffic sign recognition for autonomous vehicles. -->
		</p>
		<p style="color:white;">
		My research focuses on bioinformatics, machine learning, optimization, and statistical modeling. I specialize in designing data analysis pipelines for biological research projects, and developing machine learning solutions for extracting meaningful signals from large-scale omics data.
		</p>
		
		<p style="color:white;">
		Recently, I've been applying Large Language Models (LLMs) to solve biological problems by leveraging domain knowledge to guide the model towards learning biologically plausible predictions. 
		</p>
		
		<p style="color:white;"> 
		I have collaborated with institutions like Stanford University and Cincinnati Children’s Hospital, with my work published in top journals such as Neuron, Cell Stem Cell, and Immunity. Notably, my co-first author paper in Neuron was the best paper of 2019.
		</p>		



		<font color= #D1F078>I am currently on the job market. </font> 
		<a href="data/Resume_Zuolin_V9.6_.pdf">[Resume]</a>
		<a href="https://scholar.google.com/citations?user=6pHncW4AAAAJ&hl=en&oi=ao">[Google Scholar]</a>
		<a href="https://github.com/yu-lab-vt/CMC">[GitHub<small>(YuLab)</small>]</a>
		
		</br>Please feel free to contact me (<a href="mailto:zuolin8@vt.edu">zuolin8@vt.edu</a>) if you’re interested.
						
		</div>
	</div>
	</div>
</div>
       

<style>

</style>






<br> <!-- Adding an empty line -->



<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Ongoing Projects</title>
    <style>
        body {
            background-color: black;
            color: white;
            font-family: Arial, sans-serif;
        }
        .container {
            width: 80%;
            margin: 0 auto;
            padding: 20px;
        }
        .section-title {
            font-size: 2em;
            margin-bottom: 20px;
        }
        .project {
            margin-bottom: 20px;
        }
        .project-title {
            font-size: 1.2em;
            margin-bottom: 5px;
        }
        .project-details {
            list-style-type: disc;
            font-size: 18px;
            margin-left: 0px;
        }
        
        .half-empty-line {
            margin: 0;
            padding: 0;
            height: 0.5em; /* Adjust the height as needed */
        }
        
    </style>
</head>


<!-- News -->
<body>
    <div class="container">
        <section id="News" class="projects-section"  align="justify">
            <h2 class="section-title">Recent News</h2>
            <ul class="project-details">
            	<p><li><b>2024-05-31</b>: Our paper, “An Inducible Genetic Tool for Tracking and Manipulating Specific Microglial States in Development and Disease,” 
            	is published in <b>Immunity</b>.</li></p>
            	<p><li><b>2024-04-20</b>: Our paper, “An Efficient and Principled Model to Jointly Learn the Agnostic and Multifactorial Effect in Large-Scale Biological Data,” 
            	is now available on bioRxiv.</li></p>
            	<p><li><b>2024-03-22</b>: The R package of our tool, “CMC-GOTermActivity,” has been updated to version 0.3.1 on <a href="https://github.com/yu-lab-vt/CMC/tree/CMC-GOTermActivity">GitHub</a>. 
            	This new release includes a comprehensive 
            	user manual <a href="data/GOTermActivity_manual_V0.3.1.pdf">(link)</a>
            	 and is now compatible with the latest version (v5) of the popular scRNA-seq analysis package, Seurat.</li></p>          	
            </ul>
        </section>
    </div>
</body>




<!-- Ongoing Projects -->
<body>
    <div class="container">
        <section id="OngoingProjects" class="projects-section"  align="justify">
            <h2 class="section-title">Ongoing Projects</h2>
            <div class="project">
                <h3 class="project-title">GO-BERT: Cell Type Annotation for scRNA-seq Data Using Large Language Model (LLM)</h3>
                <ul class="project-details">
                    <li>Leveraged domain knowledge from Gene Ontology (GO) to enhance the model architecture, guiding the model towards biologically plausible predictions in cell-type annotations.</li>
                    <li>Working on improving model’s awareness of cell’s global context to ensure more precise predictions.</li>
                </ul>
            </div>
            <div class="project">
                <h3 class="project-title">Peak Calling Analysis for CUT&amp;RUN Chromatin Profiling Data</h3>
                <ul class="project-details">
                    <li>Developing a statistical model to robustly identify DNA binding sites, tailored specifically for the unique characteristics of CUT&amp;RUN chromatin profiling data.</li>
                </ul>
            </div>
            <div class="project">
                <h3 class="project-title">Therapy Data Analysis for Non-Small Cell Lung Cancer Patients</h3>
                <ul class="project-details">
                    <li>Developed a treatment effectiveness analysis pipeline, identifying two patient clusters with distinct response mechanisms to "chemotherapy + immunotherapy."</li>
                </ul>
            </div>
        </section>
    </div>
</body>




<!-- Completed Projects -->
<div id="SelectedProjects" class="page-header container">
	<h1  style="color:white;">Selected Projects</h1>
</div>	

<div class="container">	

    <!-- Neuron -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;"> Heterogeneity of Microglia and Brain Myeloid Cells During Development</h2>
        <div class="project-description"> Designed and implemented scRNA-seq data analysis pipeline. 
        <a href="https://www.sciencedirect.com/science/article/pii/S0896627318310821?via%3Dihub">  [Paper: <span class="smaller-text">Neuron</span>]</a><p>
        </div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive" src="data/imgs/Neuron_P1.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><b>scRNA-seq Analysis pipeline</b> (alignment, QC, normalization, clustering, DEG analysis, etc.)</p>
            </div>
        </div>
        
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive" src="data/imgs/Neuron_P2_2.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>Pseudotime analysis;<br>Pathway/GO enrichment analysis;<br>Regulation network construction;</p>
            </div>
        </div>

        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive" src="data/imgs/Neuron_P3_2.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><b>Cell cycle reconstruction; <br> Cell types similarity analysis</b> (newly-found PAM vs. well-know DAM);</p>
            </div>
        </div>
        
    </div>
    <br><br> 
    


    <!-- CMC Project -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;">CMC: A Machine Learning Model Unifying Various Omics Analysis Tasks</h2>
        <div class="half-empty-line"></div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/CMC.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>CMC advances various omics analysis tasks by transforming complex tasks into a unified mathematical problem: inferring the effect of multiple hidden factors in large-scale omics data.
				</br>
				<a href="https://github.com/yu-lab-vt/CMC">[GitHub: <span class="smaller-text">CMC</span>]</a> 
				<a href="https://www.biorxiv.org/content/10.1101/2024.04.12.589306v1.full.pdf"> [Paper: <span class="smaller-text">CMC</span>]</a></p>
            </div>
        </div>
    </div>
    <br><br>
	
	
    <!-- App1: Driving TF -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;">(CMC Application 1) Driving Transcription Factor(TF) Identification</h2>
        <div class="project-description" align="justify"> Driving TF identification aims to pinpoint TFs responsible for biological processes or diseases by mining a large-scale TF-gene binding database. 
        <a href="https://github.com/yu-lab-vt/CMC/tree/CMC-DrivingTFDetection">[GitHub: <span class="smaller-text">DrivingTF</span>]</a> 
        <br> Using CMC, we systematically models gene & TF binding affinity heterogeneity, leading to more accurate detection.</p>
        </div>	
        
        <div class="project-content" >
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/DrivingTF_P1.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>We observed significant heterogeneity in gene and TF binding affinities.</p>
            </div>
        </div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/DrivingTF_P2.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><br>The model successfully identified Zfp36l1 as a key regulator of glial fate specification and glioma growth, confirmed by knock-out experiments, leading to genuine biological discoveries.
                </br><a href="https://www.sciencedirect.com/science/article/pii/S1934590919301055"> [Paper: <span class="smaller-text">DrivingTF</span>]</a></p>
            </div>
        </div> 
    </div>
    <br><br>
    
    
    <!-- Application 2:scRNA-seq Normalization -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;">(CMC Application 2) scRNA-seq Data Normalization</h2>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/scRNANorm.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
            	<p><a href="https://github.com/yu-lab-vt/CMC/tree/CMC-3DNorm">[GitHub: <span class="smaller-text">scRNAseq_Norm</span>]</a>
            	<br><br><br><br><br>
                <p>We observed that “effective cDNA-length” significantly impacts scRNA-seq raw counts.</p>
            </div>
        </div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/scRNANorm_P2.png" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>CMC-based normalization reduces variances of housekeeping genes, while preserving variances of differentially expressed genes (DEGs).</p>
            </div>
        </div>
    </div>
    <br><br>
    
    
    
    <!-- Application 3:TySim -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;"> (CMC Application 3) TySim: Quantifying Similarity of Single Cells to Target Cell Type</h2>
        <div class="project-description"> TySim is the first tool to quantify single-cell similarity to a target cell type (for scRNA-seq data across different sequencing platforms) 
        <a href="https://ieeexplore.ieee.org/abstract/document/9995574">[Paper: <span class="smaller-text">TySim</span>]</a> 
        <a href="https://github.com/yu-lab-vt/CMC/tree/CMC-TySim">  [GitHub: <span class="smaller-text">TySim</span>]</a><p>
        </div>
        
        <div class="project-description"> It is developed to address biologists' questions such as:<br>
			(1) What is the relationship between a newly discovered cell type and a known cell type?<br>
			(2) Does a cell type of interest exist in my single cell population?</p>	
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/TySim_P1.svg" alt="CMC Project Image">
            </div>   
        </div>
        </div>
        
        <br>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/TySim_P2.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>TySim enables the identification of cell type that was not discovered due to contamination in scRNA-seq data.</p>
            </div>
        </div>
        
        <br>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/TySim_P3.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>TySim was applied to validate the effectiveness of an inducible genetic tool for tracking microglial subtypes in disease.
                <br><a href="https://www.cell.com/immunity/fulltext/S1074-7613(24)00258-9">[Paper: <span class="smaller-text">Immunity</span>]</a> </p>
            </div>
        </div>
    
    </div>
    <br><br>    
    
    
    
    	
    <!-- Application 4:GO Term -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;"> (CMC Application 4) Gene Ontology (GO) term activity score transformation</h2>
        <div class="project-description"> A model to transform the scRNA-seq dataset into GO term activity score. 
        <a href="https://github.com/yu-lab-vt/CMC/tree/CMC-GOTermActivity">  [GitHub: <span class="smaller-text">GO-Transform</span>]</a><p>
        </div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GOTerm_P1.svg" alt="CMC Project Image">
            </div>
        </div>
        
        <br>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GOTerm_P2_3.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>GO term activity transformation enhances downstream clustering analysis.</p>
            </div>
        </div>
    </div>
    <br><br>        	
    	
    	
    <!-- Application 5:CancerGene -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;"> (CMC Application 5) Cancer Associated Gene Identification</h2>
        <div class="project-description"> A model to identify the genes whose mutations are responsible for a particular cancer initiation and progression. 
        <a href="https://github.com/yu-lab-vt/CMC/tree/CMC-CancerGeneFinder">  [GitHub: <span class="smaller-text">CancerGeneFinder</span>]</a><p>
        </div>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/CancerGene_P1.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><b>Question: which gene’s mutations are responsible for cancer’s initiation & progression? </b></p>
            </div>
        </div>
        
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/CancerGene_P2.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>Mutation rates vary across patients, mutation types, and other factors.</p>
            </div>
        </div>

        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/CancerGene_P3_2.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><b>CMC reduces potential artefactual findings</b> by modeling the heterogeneity of mutation rates.</p>
            </div>
        </div>
        
    </div>
    <br><br>   
    
    
    
    <!-- Imputation -->
    <div class="project-container">
        <h2 style="color:white; width: 100%;"> GLImpute: Missing Value Imputation by Joint Inference of Global-Local Interaction Network</h2>
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GLImpute_P1.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p><b>Missing Value Imputation</b> by joint inference of Global-Local interaction network</p>
            </div>
        </div>
        
        <br>
        
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GLImpute_P2.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>GLImpute achieved significantly lower imputation errors regardless of local/global component ratio.</p>
            </div>
        </div>
        
        <br>
        
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GLImpute_P3.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>Superior performance of GLImpute on metabolite data</p>
            </div>
        </div>
        
        <br>
        
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GLImpute_P5.svg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>GLImpute consistently outperforms others in imputing traffic speed data over 61 days</p>
            </div>
        </div>
        
        <br>
                
        <div class="project-content">
            <div class="project-image">
                <img class="img-responsive img-rounded" src="data/imgs/GLImpute_P4_2.jpg" alt="CMC Project Image">
            </div>
            <div class="project-description">
                <p>GLImpute well recovers genes’ expression patterns in scRNA-seq data</p>
            </div>
        </div>        
                
    </div>
    <br><br> 
        
    
    
	
</div>




<style>
    .project-container {
        width: 100%;
        color: white;
    }
    .project-content {
        display: flex;
        flex-wrap: nowrap;
        align-items: flex-start;
    }
    .project-image {
        flex: 0 0 70%;
        padding-right: 15px;
    }
    .project-image img {
        width: 100%;
        height: auto;
        display: block;
        margin-top: 0;
    }
    .project-description {
        flex: 0 0 30%;
        margin-top: 0;
        font-size: 18px; /* Increased font size */
        line-height: 1.5; /* Adjusted line height for better readability */
    }
    .project-description p:first-child {
        margin-top: 0;
    }
    .project-description a {
        font-size: 18px; /* Slightly smaller font size for links */
    }
    .project-description .smaller-text {
        font-size: 13px; /* Smaller font size for the CMC part */
    }
</style>






<!-- Publications -->
<div id="Publications" class="page-header container">
	<h1 style="color:white;">Publications</h1>
  </div>
<div class="container">

	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Developmental heterogeneity of microglia and brain myeloid cells revealed by deep single-cell RNA sequencing</b><br>
		Li Q*, <b>Cheng Z* (co-first author)</b>, Zhou L, Darmanis S, Neff NF, Okamoto J, … & Tony Wyss-Coray, Ben A Barres<br>
		<i><b>Neuron</b>, Jan. 2019.</i><br>
		<a href="https://www.sciencedirect.com/science/article/pii/S0896627318310821"> <small>[Paper] </small></a>
		<!-- <a href="https://github.com/yu-lab-vt/CINDA"> <small>[Code] </small></a> -->
		</div>
	</div><br>
	

	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Single-cell transcriptomics uncovers glial progenitor diversity and cell fate determinants during development and gliomagenesis </b><br>
		Weng Q, Wang J, Wang J, He D, <b>Cheng Z</b>, Zhang F, Verma R, Xu L, Dong X, Liao Y, He X, Potter A, Zhang L, Zhao C, et al.<br>
		<i><b>Cell stem cell</b>, May 2019.</i><br>
		<a href="https://www.sciencedirect.com/science/article/pii/S1934590919301055"> <small>[Paper] </small></a>
		<!-- <a href="https://github.com/yu-lab-vt/CINDA"> <small>[Code] </small></a> -->
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>An inducible genetic tool for tracking and manipulating specific microglial states in development and disease</b><br>
		Barclay K, Nora A, <b>Cheng Z</b>, Kim M, Zhou L, Yang J, Rustenhoven J, Mazzitelli J, Smyth LC, Kapadia D, Brioschi S, et al.<br>
		<i><b>Immunity</b>, May 2024.</i><br>
		<a href="https://www.cell.com/immunity/fulltext/S1074-7613(24)00258-9"> <small>[Paper] </small></a>
		<!-- <a href="https://github.com/yu-lab-vt/CINDA"> <small>[Code] </small></a> -->
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>An Efficient and Principled Model to Jointly Learn the Agnostic and Multifactorial Effect in Large-Scale Biological Data</b><br>
		<b>Cheng Z</b>, Wei S, Wang Y, Wang Y, Lu R, Wang Y, Yu G<br>
		<i>bioRxiv, Apr. 2024.</i><br>
		<a href="https://www.biorxiv.org/content/10.1101/2024.04.12.589306v1.full.pdf"> <small>[Paper]</small></a>
		<a href="https://github.com/yu-lab-vt/CMC"> <small>[Code] </small></a>
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>A Single-Cell-Resolution Quantitative Metric of Similarity to a Target Cell Type for scRNA-seq Data</b><br>
		<b>Cheng Z</b>, Wei S, Yu G<br>
		<i>In 2022 IEEE International Conference on Bioinformatics and Biomedicine (BIBM), Dec. 2022.</i><br>
		<a href="https://ieeexplore.ieee.org/abstract/document/9995574"> <small>[Paper]</small></a>
		<a href="https://github.com/yu-lab-vt/CMC/tree/CMC-TySim"> <small>[Code] </small></a>
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Self-heating burn-in pattern generation based on the genetic algorithm incorporated with a BACK-like procedure</b><br>
		<b>Cheng Z</b>, Cui X, Cui X, Lee CL<br>
		<i>IET Computers & Digital Tech., Nov. 2015.</i><br>
		<a href="https://ietresearch.onlinelibrary.wiley.com/doi/full/10.1049/iet-cdt.2014.0219"> <small>[Paper]</small></a>
		</div>
	</div><br>
	
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Cosbin: cosine score-based iterative normalization of biologically diverse samples</b><br>
		Wu CT, Shen M, Du D, <b>Cheng Z</b>, Parker SJ, Lu Y, Van Eyk JE, Yu G, Clarke R, Herrington DM, Wang Y, et al.<br>
		<i>Bioinformatics Advances, Oct. 2022.</i><br>
		<a href="https://academic.oup.com/bioinformaticsadvances/article/2/1/vbac076/6764617"> <small>[Paper]</small></a>
		<a href="https://github.com/MinjieSh/Cosbin"> <small>[Code] </small></a>
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Microglial microRNAs mediate sex-specific responses to tau pathology</b><br>
		Kodama L, Guzman E, Etchegaray JI, Li Y, Sayed FA, Zhou L, Zhou Y, Zhan L, Le D, Udeochu JC, Clelland CD, <b>Cheng Z</b>, et al.<br>
		<i><b>Nature Neuroscience</b>, Feb. 2020.</i><br>
		<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7394069/"> <small>[Paper] </small></a>
		</div>
	</div><br>
		
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>AD-linked R47H-TREM2 mutation induces disease-enhancing microglial states via AKT hyperactivation</b><br>
		Sayed FA, Kodama L, Fan L, Carling GK, Udeochu JC, Le D, Li Q, Zhou L, Wong MY, Horowitz R, Ye P, Mathys H, Wang M, Niu X, 
		Mazutis L, Jiang X, Wang X, Gao F, Brendel M, Maria T, Tracy TE, Frost G, Zhou Y, Li Y, Qiu Y, <b>Cheng Z</b>, Yu G, et al.<br>
		<i><b>Science Translational Medicine</b>, Dec. 2021.</i><br>
		<a href="https://www.science.org/doi/full/10.1126/scitranslmed.abe3947"> <small>[Paper] </small></a>
		</div>
	</div><br>
	
	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Paternal malnutrition programs breast cancer risk and tumor metabolism in offspring</b><br>
		da Cruz RS, Carney EJ, Clarke J, Cao H, Cruz MI, Benitez C, Jin L, Fu Y, <b>Cheng Z</b>, Wang Y, de Assis S<br>
		<i>Breast Cancer Research, Dec. 2018.</i><br>
		<a href="https://link.springer.com/article/10.1186/s13058-018-1034-7"> <small>[Paper] </small></a>
		</div>
	</div><br>		
	
</div>



<!-- Award -->
<div id="Awards" class="page-header container">
	<h1 style="color:white;">Selected Awards</h1>
  </div>
<div class="container">

	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Best papers</b> 2018 – 2019 published in <b>Neuron</b> (1 of the 11)
		</br>
		</div>
	</div><br>
</div>


<!-- ACADEMIC SERVICE -->
<div id="AcademicService" class="page-header container">
	<h1 style="color:white;">Academic Service</h1>
  </div>
<div class="container">

	<div class="row">
		<div class="col-md-12" style="color:white;">
		<b>Peer Review:</b><br>
		Bioinformatics Advances; BMC bioinformatics; Scientific Reports; IET Computers & Digital Tech.<br>
		<div class="half-empty-line"></div>
		<b>Teaching Assistant:</b><br>
		Advanced Machine Learning (Fall 2021)<br>
		</div>
	</div><br><div class="half-empty-line"></div>
</div>



<!-- Education -->

<!--
<div id="Education" class="page-header container">
	<h1>Education </h1>
  </div>
<div class="container">		
	<!-- </div><hr> -->	
	
<!--	
	<div class="row">
		<div class="col-md-2">
			<a href="data/imgs/embryo_zoomin_c.png" target="_blank">
			<img class="img-responsive img-rounded" src="data/imgs/embryo_zoomin_c.png" alt="" style="text-align: center;"></a>
		</div>

		<div class="col-md-10" align="justify" >
		<b>Cell Tracking in Crowd Scenes with Deficient Detection and Segmentation Results. (10TB data)</b> <a href="https://github.com/ccwang92/embryoTracker"> [Code] </a> <br>
		Recent years have seen significant breakthroughs in microscopy techniques and fluorescence probes, which enables recording the mouse embryogenesis at cellular level for days – from a single cell to the a fully formed organism - easily generating terabyte-level 3D time-lapse data. As millions of cells exist in the data, it brings a natural demand of an automated computational tool for its comprehensive analysis. At current stage, we are focusing on four key tasks: cell segmentation, cell tracking, data curation and visualization.
		</div>
	</div>

</div>
-->




<div id="Contact" class="page-header container">
	<h1 style="color:white;">Contact me! </h1>
  </div>
<div class="container" style="color:white;">
	
	<a href="mailto:zuolin8@vt.edu">zuolin8@vt.edu</a><br>
	Virginia Tech Research Center - Arlington<br>
	<hr>
</div>


<div class="container" style="color:white;">
<a href="https://hits.seeyoufarm.com"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fzuolincheng.github.io&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false"/></a>
</div>	


	
<!-- <footer class="footer">
	  <div class="container">
	  	<p style="text-align:center;"><br><br><img  align="middle" src="./Elahe Soltan_files/cscoated_cmyk_full.jpg" alt="" style="max-height: 160px; max-width: 200px"></p>
		
	  </div>
</footer>
<script type="text/javascript"  async="" src="./files/analytics.js"></script><script type="text/javascript" src="./files/jquery.min1.js"></script>
	<script src="./files/bootstrap.min.js"></script>
	<script>
	  $('body').scrollspy({ target: '#navbar-top', offset: 50 })
	</script>
	<script>
	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
	
	  ga('create', 'UA-64623513-1', 'auto');
	  ga('send', 'pageview');

	</script> -->

<!--Start Footer -->
			<!-- Start of StatCounter Code for Default Guide -->
<!-- <script type="text/javascript"  src="./files/exp.js"></script>

 Start of StatCounter Code for Default Guide -->
<!-- <script type="text/javascript">
var sc_project=11628203; 
var sc_invisible=1; 
var sc_security="3b92a07a"; 
</script> -->
<!-- Default Statcounter code for personal profile
http://www.cs.virginia.edu/~yj9xs/ -->
<!-- <script type="text/javascript">
var sc_project=11628203; 
var sc_invisible=1; 
var sc_security="931e1758"; 
</script>
<script type="text/javascript"
src="https://www.statcounter.com/counter/counter.js"
async></script>
<noscript><div class="statcounter"><a title="Web Analytics"
href="https:/statcounter.com/" target="_blank"><img
class="statcounter"
src="https://c.statcounter.com/11628203/0/931e1758/1/"
alt="Web Analytics"></a></div></noscript> -->
<!-- End of Statcounter Code -->
 </body></html>
